modules/styling/stories/MergingStyles.stories.mdx

```diff
@@ -1,6 +1,7 @@
 import {SymbolDoc} from '@workday/canvas-kit-docs';
 
 import {CreateStencil} from './examples/CreateStencil';
+import {StylingOverrides} from './examples/StylingOverrides';
```

Comment:
This was accidentally removed in #2498 from another file and not added here. The `ExampleCode` block below did not error, but instead could not find the story so the example rendered as "No code available" and a blank example:

![empty example](https://github.com/Workday/canvas-kit/assets/338257/25d52d33-e06b-4855-ba8a-907a4f18d12e)

Importing the example fixes the documentation.